AODN/AODN-WAVE-DM/BOM-WAVE-DM/bom_wave_library/generate_nc_file_att

```diff
@@ -2,7 +2,7 @@
 abstract                 =    
 acknowledgement          = These data were collected and Quality controlled by the Australian Bureau of Meteorology. These data were made freely available by the AODN - a data facility of the Integrated Marine Observing System [IMOS]. IMOS is supported by the Australian Government.
 author                   = Besnard, Laurent
-buoy_specification_url   =
+buoy_specification_url   = https://s3-ap-southeast-2.amazonaws.com/content.aodn.org.au/Documents/AODN/Waves/Instruments_manuals/datawell_brochure_dwr4_acm_b-38-09.pdf
```

Comment:
you can replace  `https://s3-ap-southeast-2.amazonaws.com/http://content.aodn.org.au/` by `http://content.aodn.org.au/`